### DISCUSSION TITLE
Any idea if Coral in M.2 slot consumes SATA port?

### DISCUSSION THREAD

**Discussion by thinkloop** [blakeblackshear/frigate#5889] [Ask A Question] [answered]
I am looking to build a new system using an [AsRock Rack C246 WSI Mini-ITX Server Motherboard](https://www.asrockrack.com/general/productdetail.asp?Model=C246%20WSI#Specifications). It allows for up to eight SATA 6Gb/s connections: four on the board, and four through an OCuLINK connection. However the specs also say that: "SATA 1 will be disable when M.2 is SATA signal". If I have a [Coral TPU](https://coral.ai/products/m2-accelerator-dual-edgetpu/) in the M.2 slot, anyone have any idea if that would disable the SATA port, leaving only 7?

**Comment by NickM-27** [comment, marked as answer]
No, the coral is pcie only
